bookmarked.html

```diff
@@ -27,20 +29,22 @@ <h1 class="header__headline">Quiz App</h1>
           </svg>
         </button>
         <span>Will there be a next question?</span>
-        <button class="card__button" class="card__button_show">
+        <button class="card__button" class="card__button--show">
           Show answer
         </button>
         <!--<p class="card__answer">New answer</p>
 <button class="card__button" class="card__button_hide">Hide answer</button> -->
```

Comment:
If you want to hide the content of an HTML element for the user you can also do this by adding the attribute `hidden` to your element. :)